src/e2e/pages/dashBoardPage.ts

```diff
@@ -91,6 +100,25 @@ export class DashboardPage {
     this.aboutActiveExposuresPopup = page.getByRole("dialog", {
       name: "About your number of active exposures",
     });
+
+    this.subscribeButton = page.getByRole("button", {
+      name: "Automatic data removal: Off",
+    });
+    this.subscribeDialog = page.getByRole("dialog", {
+      name: "Turn on automatic data removal with Monitor Plus",
+    });
+    this.subscribeDialogCloseButton = page.getByRole("button", {
+      name: "Close",
+    });
+    this.subscribeDialogSelectYearlyPlanLink = page.getByRole("link", {
+      name: "Select yearly plan",
+    });
+    this.subscribeDialogSelectMonthlyPlanLink = page.getByRole("link", {
+      name: "Select monthly plan",
+    });
+    this.yearlyMonthlyTablist = page.getByText("YearlyMonthly");
```

Comment:
I see this is used to verify that the tabs are visible, but wouldn't the two selectors below this line suffice for that?

Reply:
its for a more comprehensive verification of all elements, its a good replacement for screenshots